modules/rhdh-plugins-reference/proc-setting-up-openshift-pipelines-to-view-detailed-scan-reports.adoc

```diff
@@ -0,0 +1,142 @@
+:_mod-docs-content-type: PROCEDURE
+[id="setting-up-openshift-pipelines-to-view-detailed-vulnerability-scan-reports_{context}"]
+
+= Setting up OpenShift Pipelines to view ACS and EC scan reports
+
+[IMPORTANT]
+====
+The scan tasks discussed in this chapter, specifically those using `roxctl` and `enterprise-contract`, are presented as examples. Both are community projects and, as such, are not supported by Red Hat Developer Hub by default. For guidance on implementing these or any other community-supported tasks consult your support agreements or contact support services.
+====
+
+In Red Hat Developer Hub, under the CI tab, the Pipeline Runs section offers a feature to access and interpret detailed task reports through a structured pop-ip interface. The pop-up consists of the following sections:
+
+* *Enterprise Contract (conditionally shown on the availability of EC task):* IIf you have used Enterprise Contract (EC) scan tasks in your pipeline, this section displays information regarding your application's compliance with the Supply chain Levels for Software Artifacts (SLSA) security framework. Enterprise Contract is a tool that verifies the provenance of application snapshots against a defined policy, ensuring compliance with the Supply chain Levels for Software Artifacts (SLSA) security framework.
+
+* *(Optional) Advanced Cluster Security (conditionally shown on the availability of ACS tasks):* If you have used Advanced Cluster Security (ACS) scan tasks in your pipeline, this section displays the individual tabs all the ACS scan tasks, for example, Image scan, Image check, and Deployment check, and offers an initial summary of the security issues.
```

Comment:
Can we remove `(Optional)` text here, since we have text saying it will be conditionally shown ?